hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyCreateRequest.java

```diff
@@ -93,6 +94,10 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     KeyArgs keyArgs = createKeyRequest.getKeyArgs();
 
     // Verify key name
+    if (keyArgs.getKeyName().startsWith(OM_SNAPSHOT_INDICATOR)) {
+      throw new OMException("Can not use key name starting with prefix: "
+          + OM_SNAPSHOT_INDICATOR, OMException.ResultCodes.INVALID_KEY_NAME);
```

Comment:
Yup. Need `OM_KEY_PREFIX` as well.

```suggestion
    if (keyArgs.getKeyName().startsWith(OM_SNAPSHOT_INDICATOR + OM_KEY_PREFIX)) {
      throw new OMException("Cannot create key under path reserved for snapshot path: "
          + OM_SNAPSHOT_INDICATOR + OM_KEY_PREFIX, OMException.ResultCodes.INVALID_KEY_NAME);
```

Reply:
yes sure!